app/templates/people/person/positions/minister/edit.hbs

```diff
@@ -0,0 +1,259 @@
+{{page-title "Bewerk"}}
+
+<div class="au-c-body-container au-c-body-container--scroll">
+    <div class="au-o-box au-o-flow au-o-flow--large">
+        <PageHeader>
+        <:title>
+            Positie:
+            {{@model.minister.ministerPosition.function.label}}
+            <br />
+            {{@model.minister.ministerPosition.worshipService.name}}
+        </:title>
+        <:subtitle>
+            {{@model.person.givenName}}
+            {{@model.person.familyName}}
+        </:subtitle>
+        </PageHeader>
+    
+
+        <form
+            aria-label="minister-position-edit-form"
+            {{on "submit" (perform this.save)}}
+            class="au-o-flow au-o-flow--large"
+        >
+            <DataCard>
+                <:title>Positie</:title>
+                <:card as |Card|>
+                <Card.Columns>
+                    <:left as |Item|>
+                        <Item>
+                            <:label>Rol</:label>
+                            <:content>
+                                {{this.model.minister.ministerPosition.function.label}}
+                            </:content>
+                        </Item>
+                        <Item>
+                            <:label>Type</:label>
+                            <:content>Bedienaar</:content>
+                        </Item>
+                    </:left>
+                    <:right as |Item|>
+                        <Item>
+                            <:label>Naam eredienst</:label>
+                            <:content>
+                                {{this.model.minister.ministerPosition.worshipService.name}}
+                            </:content>
+                        </Item>
+                    </:right>
+                </Card.Columns>
+                </:card>
+            </DataCard>
+
+            <EditCard>
+            <:title>Primaire contactgegevens</:title>
+            <:card as |Card|>
+                <Card.Columns>
+                <:left as |Item|>
+                    {{#if this.model.minister.contacts.firstObject.contactAddress.fullAddress}}
+                    <Item>
+                        <:label>Full address</:label>
+                        <:content>
+                        {{this.model.minister.contacts.firstObject.contactAddress.fullAddress}}
+                        </:content>
+                    </Item>
+                    {{/if}}
```

Comment:
I think this can be removed. The new data should have proper address information instead of only the fullAddress attribute.

Reply:
Same as before. I did to make it consistent with the mandataris edit page.